extensions/Worker.Extensions.CosmosDB/src/Config/CosmosDBBindingOptionsSetup.cs

```diff
@@ -15,10 +16,18 @@ internal class CosmosDBBindingOptionsSetup : IConfigureNamedOptions<CosmosDBBind
         private readonly IConfiguration _configuration;
         private readonly AzureComponentFactory _componentFactory;
 
-        public CosmosDBBindingOptionsSetup(IConfiguration configuration, AzureComponentFactory componentFactory)
+        public CosmosDBBindingOptionsSetup(IOptions<WorkerOptions> workerOptions, IOptions<CosmosDBBindingOptions> cosmosOptions, IConfiguration configuration, AzureComponentFactory componentFactory)
```

Comment:
```suggestion
        public CosmosDBBindingOptionsSetup(IOptions<WorkerOptions> workerOptions, IConfiguration configuration, AzureComponentFactory componentFactory)
```

This type is configuring `CosmosDBBindingOptions`, it cannot import it. Save `IOptions<WorkerOptions>` to a field and set the serializer as part of the `Configure` call.

Reply:
That's the whole point of this class, how did I miss that 🤦🏻‍♀️ 🤦🏻‍♀️